examples/server_client.rs

```diff
@@ -9,7 +9,7 @@ use std::thread;
 const SERVER: &str = "127.0.0.1:12351";
 
 fn server() -> Result<(), ErrorKind> {
-    let (mut socket, packet_sender, event_receiver) = Socket::bind(SERVER, Config::default())?;
+    let (mut socket, packet_sender, event_receiver) = Socket::bind(SERVER)?;
     let _thread = thread::spawn(move || socket.start_polling());
 
     println!("Listening for connections to {}", SERVER);
```

Comment:
We should probably use a logging facility for this, `info!()` for example

Reply:
It is not related to my PR but I can throw it in.

Reply:
Apologies, didn't even think about that. It's fine with me with either way. Resolving this.